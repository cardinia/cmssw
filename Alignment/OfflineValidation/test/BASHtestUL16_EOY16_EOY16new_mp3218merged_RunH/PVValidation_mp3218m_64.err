26-Sep-2019 12:51:57 CEST  Initiating request to open file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/B2E7C156-CE11-E911-92BC-0CC47A78A4B0.root
26-Sep-2019 12:52:00 CEST  Successfully opened file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/B2E7C156-CE11-E911-92BC-0CC47A78A4B0.root
Begin processing the 1st record. Run 283415, Event 72170, LumiSection 1 on stream 0 at 26-Sep-2019 12:52:09.117 CEST
Begin processing the 1001st record. Run 283415, Event 13075409, LumiSection 5 on stream 0 at 26-Sep-2019 12:53:59.537 CEST
Begin processing the 2001st record. Run 283548, Event 282801590, LumiSection 287 on stream 0 at 26-Sep-2019 12:54:11.051 CEST
Begin processing the 3001st record. Run 283675, Event 18391590, LumiSection 9 on stream 0 at 26-Sep-2019 12:54:17.077 CEST
Begin processing the 4001st record. Run 283820, Event 2689474524, LumiSection 1531 on stream 0 at 26-Sep-2019 12:54:19.617 CEST
Begin processing the 5001st record. Run 283820, Event 2698062344, LumiSection 1535 on stream 0 at 26-Sep-2019 12:54:21.011 CEST
Begin processing the 6001st record. Run 283835, Event 8499120, LumiSection 5 on stream 0 at 26-Sep-2019 12:54:22.236 CEST
Begin processing the 7001st record. Run 283835, Event 14082309, LumiSection 9 on stream 0 at 26-Sep-2019 12:54:23.235 CEST
Begin processing the 8001st record. Run 283835, Event 22655685, LumiSection 14 on stream 0 at 26-Sep-2019 12:54:24.282 CEST
Begin processing the 9001st record. Run 283885, Event 2989114072, LumiSection 1716 on stream 0 at 26-Sep-2019 12:54:25.233 CEST
Begin processing the 10001st record. Run 283885, Event 2996462832, LumiSection 1720 on stream 0 at 26-Sep-2019 12:54:26.284 CEST
Begin processing the 11001st record. Run 284006, Event 534657684, LumiSection 369 on stream 0 at 26-Sep-2019 12:54:27.396 CEST
Begin processing the 12001st record. Run 284006, Event 543307489, LumiSection 374 on stream 0 at 26-Sep-2019 12:54:28.714 CEST
Begin processing the 13001st record. Run 284006, Event 549827385, LumiSection 378 on stream 0 at 26-Sep-2019 12:54:29.955 CEST
Begin processing the 14001st record. Run 284006, Event 557060420, LumiSection 382 on stream 0 at 26-Sep-2019 12:54:31.191 CEST
Begin processing the 15001st record. Run 284042, Event 199786541, LumiSection 123 on stream 0 at 26-Sep-2019 12:54:32.371 CEST
Begin processing the 16001st record. Run 284042, Event 206899988, LumiSection 128 on stream 0 at 26-Sep-2019 12:54:33.708 CEST
26-Sep-2019 12:54:33 CEST  Closed file root://eoscms.cern.ch//eos/cms/store/data/Run2016H/HLTPhysics/ALCARECO/TkAlMinBias-07Dec2018-v1/20000/B2E7C156-CE11-E911-92BC-0CC47A78A4B0.root

=============================================

MessageLogger Summary

 type     category        sev    module        subroutine        count    total
 ---- -------------------- -- ---------------- ----------------  -----    -----
    1 fileAction           -s file_close                             1        1
    2 fileAction           -s file_open                              2        2

 type    category    Examples: run/evt        run/evt          run/evt
 ---- -------------------- ---------------- ---------------- ----------------
    1 fileAction           PostGlobalEndRun                  
    2 fileAction           pre-events       pre-events       

Severity    # Occurrences   Total Occurrences
--------    -------------   -----------------
System                  3                   3

dropped waiting message count 0
[7.301MB/7.301MB][100%][==================================================][7.301MB/s]  [7.301MB/7.301MB][100%][==================================================][7.301MB/s]  
