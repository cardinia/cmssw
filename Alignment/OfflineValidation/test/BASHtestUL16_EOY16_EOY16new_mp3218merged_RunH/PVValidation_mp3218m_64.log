000 (3189599.064.000) 09/26 11:25:39 Job submitted from host: <137.138.44.75:9618?addrs=137.138.44.75-9618+[2001-1458-d00-1a--b]-9618&alias=bigbird14.cern.ch&noUDP&sock=schedd_2873_37ae_40>
...
040 (3189599.064.000) 09/26 12:51:08 Started transferring input files
	Transferring to host: <128.142.103.157:9618?addrs=128.142.103.157-9618+[2001-1458-301-89--100-3a0]-9618&noUDP&sock=3630_69cd_25078>
...
040 (3189599.064.000) 09/26 12:51:08 Finished transferring input files
...
001 (3189599.064.000) 09/26 12:51:08 Job executing on host: <128.142.103.157:9618?addrs=128.142.103.157-9618+[2001-1458-301-89--100-3a0]-9618&noUDP&sock=1826_1006_3>
...
006 (3189599.064.000) 09/26 12:51:17 Image size of job updated: 34512
	21  -  MemoryUsage of job (MB)
	21136  -  ResidentSetSize of job (KB)
...
006 (3189599.064.000) 09/26 12:54:54 Image size of job updated: 55544
	21  -  MemoryUsage of job (MB)
	21136  -  ResidentSetSize of job (KB)
...
040 (3189599.064.000) 09/26 12:54:54 Started transferring output files
...
040 (3189599.064.000) 09/26 12:54:54 Finished transferring output files
...
005 (3189599.064.000) 09/26 12:54:54 Job terminated.
	(1) Normal termination (return value 0)
		Usr 0 00:02:14, Sys 0 00:00:05  -  Run Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Run Local Usage
		Usr 0 00:02:14, Sys 0 00:00:05  -  Total Remote Usage
		Usr 0 00:00:00, Sys 0 00:00:00  -  Total Local Usage
	4800  -  Run Bytes Sent By Job
	1015  -  Run Bytes Received By Job
	4800  -  Total Bytes Sent By Job
	1015  -  Total Bytes Received By Job
	Partitionable Resources :    Usage  Request Allocated 
	   Cpus                 :                 1           
	   Disk (KB)            :     7519        1   1528349 
	   Memory (MB)          :       21     2000      2000 
...
