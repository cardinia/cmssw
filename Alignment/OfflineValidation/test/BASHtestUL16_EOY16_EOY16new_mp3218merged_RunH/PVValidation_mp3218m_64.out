Job started at  Thu Sep 26 12:51:09 CEST 2019
batch dir: /pool/condor/dir_25391 release: /afs/cern.ch/cms/CAF/CMSALCA/ALCA_TRACKERALIGN/data/commonValidation/alignmentObjects/paconnor/CMSSW_10_6_0/src release base: /cvmfs/cms.cern.ch/slc7_amd64_gcc700/cms/cmssw/CMSSW_10_6_0
cmsRun PVValidation_mp3218m_283415_cfg.py
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: This is DATA!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Applying TrackerSurfaceDeformations!
>>>>>>>>>> testPVValidation_cfg.py: msg%-i: Running DA Algorithm!
Content of working dir is total 7.4M -rw-r--r--. 1 tkello zh 7.4M Sep 26 12:54 PVValidation_mp3218m_283415.root -rw-r--r--. 1 tkello zh 20K Sep 26 12:51 PVValidation_mp3218m_283415_cfg.py -rw-r--r--. 1 tkello zh 3.5K Sep 26 12:54 _condor_stderr -rw-r--r--. 1 tkello zh 522 Sep 26 12:51 _condor_stdout -rwxr-xr-x. 1 tkello zh 1015 Sep 26 12:51 condor_exec.exe -rw-------. 1 tkello zh 4.3K Sep 26 12:51 tkello.cc drwx------. 3 tkello zh 4.0K Sep 26 12:51 tmp drwx------. 3 tkello zh 4.0K Sep 26 12:51 var
Job ended at  Thu Sep 26 12:54:54 CEST 2019
